File: app/src/main/java/com/example/kineduexample/entities/Articles.java
package com.example.kineduexample.entities;

import com.google.gson.annotations.SerializedName;

class Articles {
    @SerializedName("type")
    private String type;

    @SerializedName("id")
    private Integer id;

    @SerializedName("name")
    private String name;

    @SerializedName("min_age")
    private Integer minAge;

    @SerializedName("max_age")
    private Integer maxAge;

    @SerializedName("picture")
    private String picture;

    @SerializedName("area_id")
    private Integer areaId;

    @SerializedName("short_description")
    private String shortDescription;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getMinAge() {
        return minAge;
    }

    public void setMinAge(Integer minAge) {
        this.minAge = minAge;
    }

    public Integer getMaxAge() {
        return maxAge;
    }

    public void setMaxAge(Integer maxAge) {
        this.maxAge = maxAge;
    }

    public String getPicture() {
        return picture;
    }

    public void setPicture(String picture) {
        this.picture = picture;
    }

    public Integer getAreaId() {
        return areaId;
    }

    public void setAreaId(Integer areaId) {
        this.areaId = areaId;
    }

    public String getShortDescription() {
        return shortDescription;
    }

    public void setShortDescription(String shortDescription) {
        this.shortDescription = shortDescription;
    }

    @Override
    public String toString() {
        return "Articles{" +
                "type='" + type + '\'' +
                ", id=" + id +
                ", name='" + name + '\'' +
                ", minAge=" + minAge +
                ", maxAge=" + maxAge +
                ", picture='" + picture + '\'' +
                ", areaId=" + areaId +
                ", shortDescription='" + shortDescription + '\'' +
                '}';
    }
}

File: app/src/main/java/com/example/kineduexample/data/network/model/Articles.kt
package com.example.kineduexample.data.network.model

import com.google.gson.annotations.SerializedName

class Articles {
    @SerializedName("type")
    var type: String? = null

    @SerializedName("id")
    var id: Int? = null

    @SerializedName("name")
    var name: String? = null

    @SerializedName("min_age")
    var minAge: Int? = null

    @SerializedName("max_age")
    var maxAge: Int? = null

    @SerializedName("picture")
    var picture: String? = null

    @SerializedName("area_id")
    var areaId: Int? = null

    @SerializedName("short_description")
    var shortDescription: String? = null

    override fun toString(): String {
        return "Articles{" +
                "type='" + type + '\''.toString() +
                ", id=" + id +
                ", name='" + name + '\''.toString() +
                ", minAge=" + minAge +
                ", maxAge=" + maxAge +
                ", picture='" + picture + '\''.toString() +
                ", areaId=" + areaId +
                ", shortDescription='" + shortDescription + '\''.toString() +
                '}'.toString()
    }
}
